<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step0DockWidge</class>
 <widget class="QDockWidget" name="Step0DockWidge">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>562</height>
   </rect>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="windowTitle">
   <string>Load\Save DPM data</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load saved data:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="loadFile">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the saved data to be loaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCreatedOn">
      <property name="text">
       <string>Created:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelLastEditOn">
      <property name="text">
       <string>Last edit:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFolder">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load">
      <property name="text">
       <string>   LOAD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_2">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:28px; color:#222222; background-color:#f8f9fa;&quot;&gt;The data will be permanently removed and will not be recoverable.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>6000</number>
      </property>
      <property name="text">
       <string>   delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save District Planning Module state:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you choose an already existing file name, an incremental version will be autamatically appended. Files are not overwritten. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>9000</number>
      </property>
      <property name="text">
       <string>Enter save name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileName">
      <property name="toolTipDuration">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>   SAVE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="invalidFileNameWarning">
      <property name="text">
       <string>(INVALID FILE NAME)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
